src/features/profile/mySecurity/components/Button/Button.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactNode } from 'react'
```

Comment:
Please use the components in common folder

Reply:
@muhadovic  this is not global component, it only uses for my security page.